README.md

```diff
@@ -18,7 +18,7 @@ nuget install Razorpay
 
 else  
 * Download Nuget package from [here](https://www.nuget.org/packages/Razorpay)
-* Package supports only .Net 4.0 and .net 4.7, Add the required version as reference to your project.
+* Package supports only .Net 4.0 and .net 4.8, Add the required version as reference to your project.
```

Comment:
We do support 4.5 & 4.7 too, right. Shall we have messaging as "`Package supports .Net 4.0 - .Net 4.8`" ?

Reply:
Added